### DISCUSSION TITLE
WireGuard

### DISCUSSION THREAD

**Discussion by leonovk** [coollabsio/coolify#1957] [Service Template Requests]
Add a service that will improve the web interface for managing the wireguard and the wireguard itself. https://github.com/wg-easy/wg-easy

**Comment by faahim** [comment]
Don't think this is already added, but would be amazing to have this ⏫

**Reply by DirectorX** [reply]
An option to create mesh network using wireguard would be amazing

**Comment by Deveopix** [comment]
Yeah, I looked and there doesn't seem to be a WireGuard option. Hopefully this gets added!

**Comment by PierBover** [comment]
This would be great to simplify connecting to non public databases

**Comment by aybasaran** [comment]
https://github.com/coollabsio/coolify/pull/3955 i've just deployed using this PR. It's working properly.

**Comment by peaklabs-dev** [comment]
This has been implemented and will be available in the upcoming release. Thank you for your patience.
